protocol/synthetix/subgraph/tests/handlePoolConfigurationSet.ts

```diff
@@ -1,8 +1,11 @@
 import { assert } from 'matchstick-as';
 import { ethereum, store } from '@graphprotocol/graph-ts';
 import { handleMarketCreated, handlePoolConfigurationSet, handlePoolCreated } from '../mainnet';
-import { createPoolConfigurationSetEvent, createPoolCreatedEvent } from './event-factories';
-import { createMarketRegisteredEvent } from './event-factories/createMarketRegisteredEvent';
+import {
+  createPoolConfigurationSetEvent,
+  createPoolCreatedEvent,
+  createMarketRegisteredEvent,
+} from './event-factories';
```

Comment:
Quite tho opposite of the original intention to have each in a separate file that I haven't had enough time to refactor last time (tho all the new code in markets follows)